README.md

```diff
@@ -17,28 +17,30 @@ Development builds are also available from [our website](https://pcsx2.net/downl
 
 ## Minimum
 * OS: Windows Vista SP2 or newer or GNU/Linux (32-bit or 64-bit)
-* CPU: Any that supports SSE2 (Pentium 4 and up, Athlon64 and up)
+* CPU: Any that supports SSE2 (Pentium 4 and up, Athlon64 and up) @ 1600 STR or better
 * GPU: DirectX 10 GPU or better
 * RAM: 2GB or more
 
 ## Recommended
 * OS: Windows 7/8/8.1/10 (64-bit) or GNU/Linux (64-bit)
-* CPU: Intel Haswell (or AMD equivalent) @ 3.2GHz or better
+* CPU: Intel Haswell (or AMD equivalent) @ 2000 STR or better
 * GPU: DirectX 11 GPU or greater
 * RAM: 4GB or more
 
 ## Notes
 
 - You need the [Visual C++ 2015 x86 Redistributables](https://www.microsoft.com/en-us/download/details.aspx?id=48145) for this version to work.
 
-- PCSX2 1.4.0 is the last version to support Windows XP. Windows XP is no longer getting updates (including security-related updates), and graphics drivers for Windows XP are older and no longer maintained.
+- PCSX2 1.4.0 is the last version to support Windows XP and Direct3D9. Windows XP is no longer getting updates (including security-related updates), and graphics drivers for Windows XP are older and no longer maintained.
 
 - Make sure to update your operating system, drivers, and DirectX (if applicable) to ensure you have the best experience possible. Having a newer GPU is also recommended so you have the latest supported drivers.
 
 - Because of copyright issues, and the complexity of trying to work around it, you need a BIOS dump extracted from a legitimately-owned PS2 console to use the emulator. For more information about the BIOS and how to get it from your console, visit [this page](https://pcsx2.net/config-guide/official-english-pcsx2-configuration-guide.html#Bios).
 
 - PCSX2 mainly takes advantage of 2 CPU cores. As of [this commit](https://github.com/PCSX2/pcsx2/commit/ac9bf45) PCSX2 can now take advantage of more than 2 cores using the MTVU speedhack. This can be a significant speedup on CPUs with 3+ cores, but it may be a slowdown on GS-limited games (or on CPUs with fewer than 2 cores).
 
+- You can visit [this page](https://www.cpubenchmark.net/singleThread.html) to find out your CPU's STR (Single Threaded Rating).
```

Comment:
Oh fuck, I didn't look at this.

Reply:
What about a hyperlink to the original text in the first place or mention about a reference on it to this content?

Reply:
Since there's the possibility that people might overlook this part.

Reply:
Agreed. I would _open up_ the requirements section with a note in italics something like

> _Requirements benchmarks are based on a statistic from the Passmark CPU bench marking software. When we say "STR", we are referring to Passmark's "Single Thread Rating" statistic. You can look up your CPU on https://cpubenchmark.net to see how it compares to PCSX2's requirements._

More or less. Probably want to be a bit less wordy.

Reply:
Italic isn't needed but I'll go with this suggestion instead.